app/src/main/java/com/keylesspalace/tusky/components/drafts/DraftMediaAdapter.kt

```diff
@@ -0,0 +1,81 @@
+/* Copyright 2020 Tusky Contributors
+ *
+ * This file is a part of Tusky.
+ *
+ * This program is free software; you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Tusky is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+ * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with Tusky; if not,
+ * see <http://www.gnu.org/licenses>. */
+
+package com.keylesspalace.tusky.components.drafts
+
+import android.view.ViewGroup
+import android.widget.ImageView
+import androidx.appcompat.widget.AppCompatImageView
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.engine.DiskCacheStrategy
+import com.keylesspalace.tusky.R
+import com.keylesspalace.tusky.db.DraftAttachment
+
+class DraftMediaAdapter(
+        private val attachmentClick: () -> Unit
+) : ListAdapter<DraftAttachment, DraftMediaAdapter.DraftMediaViewHolder>(
+        object: DiffUtil.ItemCallback<DraftAttachment>() {
+            override fun areItemsTheSame(oldItem: DraftAttachment, newItem: DraftAttachment): Boolean {
+                return oldItem == newItem
```

Comment:
Should this compare ids instead?

Reply:
they don't have ids, I think it is fine like this

Reply:
Oops, I confused it with something else then